.github/workflows/ci.yml

```diff
@@ -15,9 +15,9 @@ jobs:
           - stable
         mcu:
           - stm32h503
-          - stm32h562
-          - stm32h563
-          - stm32h573
+          # - stm32h562
+          # - stm32h563
+          # - stm32h573
```

Comment:
This was disappointing to disable so soon... I had forgotten that when I did the RCC in stm32-rs last year, I only focussed on the STM32H503 RCC module because of how much there is to it, and how different the STM32H562/63/73 were. I'll have to get back to that to be able to get this code working on the rest of the family.